localization/english/wc_game_concepts_magic_l_english.yml

```diff
@@ -0,0 +1,6 @@
+﻿l_english:
+ game_concept_wc_mana:0 "Mana"
+ game_concept_wc_mana_desc:0 "Mana is a resource that represents a Character's ability to bend reality to their will.\nIt can be spend on... currently nothing. Change that and this line later."
```

Comment:
I would mention that unspent Mana goes to prowess. It's important to mention for the understanding of this.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/120909273-783a4700-c684-11eb-801a-f24425d34749.png)

</details>

Reply:
Done?